Test/Utils/ComputeNode/Initialize.ps1

```diff
@@ -2,21 +2,54 @@
 
 . $PSScriptRoot\..\..\PesterLogger\PesterLogger.ps1
 . $PSScriptRoot\..\..\TestConfigurationUtils.ps1
-
+. $PSScriptRoot\Configuration.ps1
+. $PSScriptRoot\Installation.ps1
+. $PSScriptRoot\Service.ps1
 . $PSScriptRoot\..\DockerNetwork\DockerNetwork.ps1
 
 function Initialize-ComputeNode {
     Param (
         [Parameter(Mandatory=$true)] [PSSessionT] $Session,
         [Parameter(Mandatory=$true)] [TestenvConfigs] $Configs,
-        [Parameter(Mandatory=$true)] [Network[]] $Networks
+        [Parameter(Mandatory=$false)] [bool] $PrepareEnv=$true
+    )
+
+    if ($PrepareEnv) {
+        Write-Log "Installing components on testbed..."
+        Install-ComputeServices -Session $Session `
+            -SystemConfig $Configs.System `
+            -OpenStackConfig $Configs.OpenStack `
+            -ControllerConfig $Configs.Controller
+    } else {
+        Write-Log "Not performing environment setup because PrepareEnv flag is set to $PrepareEnv"
+    }
+}
+
+function Clear-ComputeNode {
+    Param (
+        [Parameter(Mandatory=$true)] [PSSessionT] $Session,
+        [Parameter(Mandatory = $true)] [SystemConfig] $SystemConfig,
+        [Parameter(Mandatory=$false)] [bool] $PrepareEnv=$true
     )
 
-    Initialize-ComputeServices -Session $Session `
-        -SystemConfig $Configs.System `
-        -OpenStackConfig $Configs.OpenStack `
-        -ControllerConfig $Configs.Controller
+    if ($PrepareEnv) {
+        Clear-TestConfiguration -Session $Session -SystemConfig $SystemConfig
+
+        Clear-Logs -LogSources (New-FileLogSource -Path (Get-ComputeLogsPath) -Sessions $Session)
+
+        Write-Log "Uninstalling components from testbed..."
+        Uninstall-Components -Session $Session
+    } else {
+        Write-Log "Not performing environment cleanup because PrepareEnv flag is set to $PrepareEnv"
+    }
+}
 
+function Initialize-DockerNetworks {
```

Comment:
This function feels like it doesn't belong here. It's kind of test-case specific. Maybe a separate file?